src/file.js

```diff
@@ -16,6 +17,17 @@ export function listLocalFiles (root) {
     nodir: true,
     ignore: getIgnoredPaths(root),
   })
+  .then(files =>
+    Promise.all(
+      files.map(file => bbStat(path.join(root, file))
+      .then(stats => ({file, stats})))
```

Comment:
I've had to re-read this three times to get that `bbStat(path.join(root, file)).then(stats => ({file, stats}))` is one thing. Can I suggest you either keep it in the same line or rearrange it so the scope of `file` is clearer?

```
files.map(file =>
  bbStat(path.join(root, file))
  .then(stats => ({file, stats}))
)
```